multimodal/tests/unittests/predictor/test_predictor.py

```diff
@@ -456,41 +456,29 @@ def test_model_configs():
                 "hf_text",
                 "timm_image",
                 "clip",
-                "categorical_transformer",
-                "numerical_transformer",
+                "ft_transformer",
                 "fusion_transformer",
             ],
-            "categorical_transformer": {
-                "out_features": 192,
+            "ft_transformer": {
+                "data_types": ["numerical", "categorical"],
+                "embedding_arch": ["linear"],
                 "d_token": 192,
-                "num_trans_blocks": 0,
-                "num_attn_heads": 4,
-                "residual_dropout": 0.0,
+                "adapter_output_feature": 192,
+                "hidden_size": 192,
+                "n_blocks": 3,
+                "attention_n_heads": 8,
                 "attention_dropout": 0.2,
-                "ffn_dropout": 0.1,
-                "normalization": "layer_norm",
-                "ffn_activation": "reglu",
-                "head_activation": "relu",
```

Comment:
Looks like `head_activation ` is removed from config?

Reply:
In line 475